apps/src/redux/isRunning.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * This duck module tracks whether the current app is running or not.
+ */
+var TOGGLE_RUNNING = 'isRunning/TOGGLE_RUNNING';
+
+module.exports.default = function (state, action) {
+  var nextState = state || false;
+
+  if (action.type === TOGGLE_RUNNING) {
+    nextState = action.isRunning;
```

Comment:
`toggleRunning` is actually the wrong name for this action - what you're doing here is `setRunning`.  Maybe should rename.